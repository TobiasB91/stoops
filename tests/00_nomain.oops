CLASS Main IS
END CLASS
