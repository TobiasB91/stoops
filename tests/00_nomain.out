Fehler: Fehlende Deklaration von main
